.claude/commands/release.md

```
@@ -117,9 +117,9 @@ STOP here. Do not push. Do not create a PR. Tell the user:
 > git push -u origin release-{VERSION}
 > ```
 >
-> Then create a PR targeting the `release-X.Y.Z` branch on Kuadrant/mcp-gateway:
+> Then create a PR targeting the `release-X.Y` branch on Kuadrant/mcp-gateway:
 > ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)